nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/operation/HotReloadStudioSnapshot.java

```diff
@@ -171,6 +173,12 @@ public Blob hotReloadPackage(DownloadablePackage remotePkg) {
             }
         }
 
+        boolean useFormerReload = Framework.isBooleanPropertyTrue(ReloadService.USE_COMPAT_HOT_RELOAD);
+        if (!useFormerReload) {
+            log.info("Use new hot reload mechanism");
```

Comment:
You may want to rename the local variable `useFormerReload` too (here and in other files), and avoid mentioning "new" in the log.